### ISSUE TITLE
Support for Pinia Vue Store

### ISSUE THREAD

**Issue by matijaparavac** [ZachJW34/nx-plus#230]
## Description
Add support for Pinia like there already is for vuex

## Motivation
Pinia is officially recomended by Vue core team

**Comment by ZachJW34** [comment]
Would you be interested in contributing a PR for a Pinia schematic? It should be very similar to the current vuex schematic in terms of implementation. I've been using Pinia and it would be a great feature to add.

**Comment by matijaparavac** [comment]
I would love to, but I have 3 kids and don't have time for anything anymore
😑
Btw, you are doing a great job with nx-plus!

**Comment by gregveres** [comment]
@ZachJW34  I am new to nx and trying to understand it. If I wanted to use pinia in my app, I could without nx-plus supporting it right? This issue is just talking about having an nx-plus command to add pinia for us right? But i could do it the normal way in each app otherwise?

**Comment by ZachJW34** [comment]
Yes you should be able to install it normally and follow the docs for getting it setup.
